core-redirects-studio-plugin/src/main/joo/com/tallence/core/redirects/studio/editor/form/RedirectEditPanel.mxml

```diff
@@ -107,22 +107,31 @@
           </plugins>
         </FieldContainer>
 
-        <ui:BoundRadioGroup width="auto"
-                            columns="1"
-                            bindTo="{getBindTo(config.localModel, RedirectImpl.SOURCE_TYPE)}"
-                            >
-          <ui:items>
-            <Radio inputValue="{RedirectImpl.SOURCE_TYPE_PLAIN}"
-                   boxLabel="{resourceManager.getString('com.tallence.core.redirects.studio.bundles.RedirectManagerStudioPlugin', 'redirectmanager_editor_field_sourceType_plain')}"/>
-            <Radio inputValue="{RedirectImpl.SOURCE_TYPE_REGEX}"
-                   boxLabel="{resourceManager.getString('com.tallence.core.redirects.studio.bundles.RedirectManagerStudioPlugin', 'redirectmanager_editor_field_sourceType_regex')}"/>
-          </ui:items>
-          <ui:plugins>
-            <ui:BindPropertyPlugin componentProperty="disabled"
+        <!--TODO both elements (spacer and comboBox) are bound to the mayUseRegexVE. There might is a more elegant way!-->
+        <editor:FormSpacerElement height="10px">
+          <editor:plugins exml:mode="append">
+            <ui:BindPropertyPlugin componentProperty="hidden"
+                                   transformer="{function(value: Boolean): Boolean {return !value;}}"
```

Comment:
Die Funktion {function(value: Boolean): Boolean {return !value;}} benutzen wir an vielen Stellen. Könnten wir diese in eine Util Klasse auslagen?

Reply:
Hab die boolean flags jetzt umbenannt, von mayWrite in mayNotWrite, dann braucht man keinen transformer mehr